### ISSUE TITLE
Architecture diagram for the package

### ISSUE THREAD

**Issue by jamesmbaazam** [epiforecasts/EpiNow2#599]

**Comment by seabbs** [comment]
Yes I think this would be amazing. @kaitejohnson as you have recently got up to speed with everything perhaps this might be something you would be keen/a good fit for?

**Comment by kaitejohnson** [comment]
Sure, though I truly hate the formatting in mermaid diagrams (last time I used them, they still didn't support latex/markdown?)
Perhaps I am out of touch. Is the EpiEstim one a mermaid diagram?

**Comment by seabbs** [comment]
Nice. Sorry I more meant use whatever you want but you could user mermaid. Point and click interfaces make me want to cry but  I assume that is a me issue.

**Comment by jamesmbaazam** [comment]
There's many good vector format options out there: figma, draw.io, etc.

**Comment by seabbs** [comment]
Done - see model overview.
